.circleci/config.yml

```diff
@@ -104,7 +104,7 @@ jobs:
     steps:
       - *attach_workspace
       - run: mkdir -p reports/junit
-      - run: npm run lint:css -- --custom-formatter node_modules/stylelint-junit-formatter > reports/junit/css-lint-results.xml
+      - run: npm run lint:css -- --custom-formatter node_modules/stylelint-junit-formatter/index.js > reports/junit/css-lint-results.xml
```

Comment:
I was able to run `npm run lint` locally with success, but once it hit CircleCI, `lint:css` would fail.

I couldn't find any documentation on it, but adding the `/index.js` to the import fixed [this](https://app.circleci.com/pipelines/github/GetJobber/atlantis/12836/workflows/8f598757-d561-402c-954e-feaf8be8c766/jobs/61067) error.